src/modules/flow/http-client/http-client.json

```diff
@@ -679,6 +719,31 @@
           "data_type": "json-array",
           "description": "The response as json-array",
           "name": "ARRAY"
+        },
+        {
+          "data_type": "boolean",
+          "description": "The response as json boolean",
+          "name": "BOOLEAN"
+        },
+        {
+          "data_type": "string",
+          "description": "The response as json string",
+          "name": "STRING"
+        },
+        {
+          "data_type": "int",
+          "description": "The response as json int",
```

Comment:
if the json object is an integer, the int packet is created with that as value and default range (min as INT32_MIN, max as INT32_MAX and step as 1)